zerver/views/user_groups.py

```diff
@@ -103,7 +103,7 @@ def update_user_group_backend(
             request, user_profile, user_group_id=user_group_id, members=delete
         ),
     ]
-    return compose_views(thunks)
+    return compose_views(request, thunks)
 
```

Comment:
Also, since `compose_views` is being used by two views should / could it be moved to be in a more general spot for helper functions?